ocs_ci/ocs/pgsql.py

```diff
@@ -132,25 +138,27 @@ def create_pgbench_benchmark(
             pg_obj = OCS(**pg_data)
             pg_obj_list.append(pg_obj)
             pg_obj.create()
-        # Confirm that expected pgbench pods are spinned
-        log.info("Searching the pgbench pods by its name pattern")
-        timeout = timeout if timeout else 300
-        for pgbench_pods in TimeoutSampler(
-            timeout, replicas, get_pod_name_by_pattern,
-            'pgbench-1-dbs-client', RIPSAW_NAMESPACE
-        ):
-            try:
-                if len(pgbench_pods) == replicas:
+
+        if wait:
+            # Confirm that expected pgbench pods are spinned
+            log.info("Searching the pgbench pods by its name pattern")
+            timeout = timeout if timeout else 300
+            for pgbench_pods in TimeoutSampler(
+                timeout, replicas, get_pod_name_by_pattern,
+                'pgbench-1-dbs-client', RIPSAW_NAMESPACE
```

Comment:
Does the patten "pgbench-1-dbs-client" remain same even after pgbench-benchmark name is provided?

Reply:
no changes for `pgbench-1-dbs-client`,  pattern will remain same